Quick_Deploy/vLLM/model_repository/vllm_model/1/model.py

```diff
@@ -0,0 +1,283 @@
+# Copyright 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
IIUC, this `model.py` is the same as we have in [vllm backend](https://github.com/triton-inference-server/vllm_backend/blob/main/src/model.py). If this is true, there is no need to keep it here, as it will make it harder to maintain this tutorial. I would suggest to revert some changes in this PR, so that vllm backend is used, and not python backend.

Reply:
ah, I see, you removed  the check for `model.json` right?